typescript-react/vitest.config.ts

```diff
@@ -1,8 +1,31 @@
+import path from 'path';
+
 import { defineConfig } from 'vitest/config';
 
 export default defineConfig({
   test: {
     globals: true,
     environment: 'jsdom',
+    setupFiles: './test.setup.ts',
+  },
+  resolve: {
+    alias: [
+      {
+        find: '@',
```

Comment:
as i like aliases i'm not a huge fan of having an alias without any keyword. I find it quite confusing and not very useful.

Reply:
Fully agree with you.
I just deleted it on both Vite configuration files `vite.config.ts` & `vitest.config.ts` and `tsconfig.json`. It's not even used on the project.